src/NBS.Appointments.Service.Core/Services/QflowService.cs

```diff
@@ -57,12 +59,41 @@ public class QflowService : IQflowService
             var policy = GetRetryPolicy();
             var response = await policy.ExecuteAsync(async (context) => {
                 query["apiSessionId"] = context["SessionId"].ToString();
-                var endpointUrl = QueryHelpers.AddQueryString($"{_options.BaseUrl}/svcCustomAppointment.svc/rest/availability", query);
+                var endpointUrl = QueryHelpers.AddQueryString($"{_options.BaseUrl}/svcCustomAppointment.svc/rest", query);
```

Comment:
Path has been changed incorrectly, availability part is required (merge issue?)

Reply:
Added validation in here as well